<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiagnosticsManufacturing::CLaSystemManufacturing</class>
 <widget class="QWidget" name="DiagnosticsManufacturing::CLaSystemManufacturing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="MainMenu::CScrollTable" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>561</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="sendTestReportBtn">
      <property name="text">
       <string>Send Test Report</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="beginTestBtn">
      <property name="text">
       <string>Begin Test</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainMenu::CScrollTable</class>
   <extends>QWidget</extends>
   <header>MainMenu/Include/ScrollTable.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
